### ISSUE TITLE
dependency-check-ant does not produce any report in 6.2.0 with default h2-Database

### ISSUE THREAD

**Issue by cstsw** [dependency-check/DependencyCheck#3415]
**Describe the bug**
In the current version 6.2.0 dependency-check-ant is unable to determine vulnerabilities or produce reports when the default h2-database is used. 

**Version of dependency-check used**
dependency-check-ant 6.2.0

**Log file**
Excerpt from running dependency-check-ant with --debug flag:

```
    [dependency-check] Unable to connect to the database
    SQLException: java.sql.SQLException: No suitable driver found for jdbc:h2:file:D:\path\to\my\dependency-check\odc;AUTOCOMMIT=ON;CACHE_SIZE=65536;RETENTION_TIME=1000;MAX_COMPACT_TIME=10000;
    	at java.sql.DriverManager.getConnection(DriverManager.java:689)
    	at java.sql.DriverManager.getConnection(DriverManager.java:247)
    	at org.owasp.dependencycheck.data.nvdcve.DatabaseManager.initialize(DatabaseManager.java:185)
    	at org.owasp.dependencycheck.data.nvdcve.DatabaseManager.<init>(DatabaseManager.java:123)
    	at org.owasp.dependencycheck.data.nvdcve.CveDB.<init>(CveDB.java:238)
    	at org.owasp.dependencycheck.Engine.openDatabase(Engine.java:987)
    	at org.owasp.dependencycheck.Engine.doUpdates(Engine.java:863)
    	at org.owasp.dependencycheck.Engine.initializeAndUpdateDatabase(Engine.java:680)
    	at org.owasp.dependencycheck.Engine.analyzeDependencies(Engine.java:606)
    	at org.owasp.dependencycheck.taskdefs.Check.callExecuteAnalysis(Check.java:1799)
    	at org.owasp.dependencycheck.taskdefs.Check.execute(Check.java:1756)
```

Aside: in contrast to version 6.1.6 the following output is missing from log produced by version 6.2.0:
```
[dependency-check] data.driver_path=''
[dependency-check] data.driver_name='org.h2.Driver
```
(These two lines appear before dependency-check connects to the database in 6.1.6.)

**To Reproduce**
1. Execute dependency-check-ant as of version 6.2.0 configured as follows:
```
<dependency-check reportoutputdirectory="${dir.reports.dc}"
                          autoupdate="true"
                          projectname="myproject"
                          reportformat="ALL"
                          datadirectory="${dependency-check.datadirectory}"
                          failonerror="false">
            <fileset dir="${basedir}/../${dir.libs}" includes="**/*.jar" />
            <fileset dir="${basedir}/.." includes="**/lib/**/*.jar" />
            <fileset dir="${basedir}" includes="**/lib/**/*.jar" />
            <suppressionfile path="${basedir}/${dir.project.build}/dependency-check-suppressions.xml" />
        </dependency-check>
```
2. See: no report is generated
(Exection does not fail due to failonerror="false")

**Expected behavior**
dependency-check-ant should be able to connect to the default h2 database.

**Additional context**

- The datadirectory mentioned in the task configuration above is shared with another project of mine that uses the Dependency-Check plugin for Gradle, currently in version 6.1.6.
- The last CVE update was executed via the Dependency-Check Gradle plugin, version 6.1.6, and is younger than 4 hours (default for cveValidForHours), so Dependency-Check Ant considers it still up to date.

**Comment by mprins** [comment]
duplicates #3416

**Comment by cstsw** [comment]
Can confirm the fix for dependency-check-ant in 6.2.1. Thanks, Jeremy!
